platform.js

```diff
@@ -201,11 +201,13 @@ module.exports = {
       });
     });
   },
-  readTextFileSync(path) {
+  readTextFileSync(path, options) {
     var stringContents = "";
+    let fsModule = fs;
```

Comment:
@tejohnso I think we should declare `electronFs` at the top of platform and do something like 
```js
stringContents = options.inASAR ? 
  electronFs.readFileSync(path, "utf8")  :
  fs.readFileSync(path, "utf8");
```

both here and in the async function that returns a promise